CIP-0XXX/README.md

```diff
@@ -0,0 +1,581 @@
+---
+CIP: ?
```

Comment:
```suggestion
CIP: 137
```

Reply:
Done in https://github.com/cardano-scaling/CIPs/commit/fe7eb48f9ddd12e226366f84871cfd019fa96559